### ISSUE TITLE
Feature request: Allow Venice to swap tiles of his puppet cities

### ISSUE THREAD

**Issue by maarten99** [LoneGazebo/Community-Patch-DLL#8699]
_1. Mod version, and DirectX version if reporting a graphical issue (X.Y.Z, e.g. 1.4.6, DX 10 & 11):_
1.4.6

_3. Error description:_ 

Right now if Venice's capital's border spreads beyond 3 tiles away (which happens often), those tiles can't be used by the capital (3 tiles away) but neither by a nearby puppet city. Allowing Venice to swap tiles would fix this.
![image](https://user-images.githubusercontent.com/24713571/155901561-0b3be22e-b26b-406b-b5f1-5bbc90e9e632.png)
![image](https://user-images.githubusercontent.com/24713571/155901564-a422b4b0-7ee9-489d-80e3-bdc04f565cf4.png)

**Comment by ilteroi** [comment]
needs to be changed in lua iirc

**Comment by LoneGazebo** [comment]
Yeah all that Venice UI exception code is kludged into the Lua all over the place.

**Comment by ilteroi** [comment]
not going to touch lua

**Comment by ilteroi** [comment]
what to do about the AI though, they are not swapping tiles at all ... i'm sort of dreaming of a system where a city can work any tile in it's working range if it's not in use by another city; independent of ownership. 

the owning city could get precedence though, if they want to work the plot the other city would have to re-allocate. but that might have a performance impact.

anyway this means no more tile swapping; less micro but the usual suspects will complain. also needs lua adaptions probably. so i don't know. maybe keep this option in mind if you tinker with the lua.
